playground/style.css

```diff
@@ -47,3 +47,41 @@ nav .btn {
   right: 0;
   height: 100% !important;
 }
+
+.examples-display {
```

Comment:
rather than doing all your own styling, can you use the bootstrap drop-down? https://getbootstrap.com/docs/4.0/components/dropdowns/

It looks like it is relatively easy to turn this into a multi-column layout: https://codepen.io/gdeichen/pen/GrEaya